apps/crm/crm/src/app/crm/components/organization/forms/organization-form/organization-form.component.ts

```diff
@@ -21,15 +21,20 @@ export class CrmOrganizationFormComponent {
   public organizationStatus = organizationStatus;
   public app = getAllAppsExcept(['crm']);
   public notifyCheckbox = new FormControl(false);
+  public currentOrgName : string;
 
   constructor(
     private organizationService: OrganizationService,
   ) {}
 
+  ngOnInit() {
+    this.currentOrgName = this.form.get('denomination').get('full').value.trim();
+  }
+
   public async uniqueOrgName() {
-    const orgName = this.form.get('denomination').get('full').value
+    const orgName = this.form.get('denomination').get('full').value.trim();
     const unique = await this.organizationService.uniqueOrgName(orgName);
-    if (!unique) {
+    if (!unique && orgName !== this.currentOrgName) {
       this.form.get('denomination').get('full').setErrors({ notUnique: true });
```

Comment:
Use changeDetectorRef so the error is displayed as soon as we leave the input
```suggestion
      this.form.get('denomination').get('full').setErrors({ notUnique: true });
      this.cdr.markForCheck();
```